frontend/components/DataFldMobile.vue

```diff
@@ -0,0 +1,64 @@
+<template>
```

Comment:
Why do you need 2 separates components for mobile and desktop? You can use media query, or `v-if` according to current device dimensions.

Reply:
Łukasz fixed this problem already, but forgot to publish the commit with deletion of this components.